src/main/webapp/js/receipt-analysis.js

```diff
@@ -17,7 +17,7 @@ function loadReceiptAnalysis() {
   const parameters = new URLSearchParams(location.search);
 
   const date = getDateFromTimestamp(parameters.get('timestamp'));
-  const storeName = parameters.get('store');
+  const storeName = capitalizeFirstLetters(parameters.get('store'));
```

Comment:
I think this is okay but will convert names like KFC, McDonald's, Chick-fil-A to Kfc, Mcdonald's and Chick-fil-a right?

Reply:
Yeah, it's the same function that's used for the home page.

Reply:
Instead of copying this function, could you reuse the one in index.js? Or move it to a common file for both files to use.

Reply:
Yeah, we were discussing creating a common file for JS functions, so I'll just move it there once we've settled on a filename.

Reply:
Sounds good. Can you add a TODO comment here for that?